src/main/java/org/example/DealershipFileManager.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class DealershipFileManager {
+    private static final String INVENTORY_FILE = "src/main/resources/inventory.csv";
+    //This method should load and read the inventory.csv
+    public void getDealership(){
+        ArrayList<Dealership> dealerships = new ArrayList<>();
+        ArrayList<Vehicle> vehicles = new ArrayList<>();
+        try(BufferedReader reader = new BufferedReader(new FileReader(INVENTORY_FILE))){
+            String line;
+            reader.readLine();
+            while((line = reader.readLine()) != null){
```

Comment:
Suggest inserting an if statement with a boolean isFirstLine, if IsFirstLine then split sentence into dealership information then set isFirstLine to false. 
Outside of if statement, continue the rest of the code to split csv into vehicle objects.